.github/workflows/main.yml

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2021 Andrew 'glyph' Reid
+#
+# SPDX-License-Identifier: Unlicense
+
 name: CI
 
```

Comment:
Oh, it's worth putting a CI job to check for licenses, just copy paste this: https://github.com/ssb-ngi-pointer/ssb-fixtures/blob/0e5992d5b5dd109ba537505091605b67b86d4b14/.github/workflows/main.yml#L14-L19